README.md

```diff
@@ -98,15 +98,131 @@ This release extends support for RHEL v8.5.
+| gfx803        | Polaris    | Polaris 20      | AMD Radeon™ PRO 575                               |  ✅   |    ✅    |   ✅    |    ✅    |    ✅    |     ✅     |     ✅     |     ✅     |     ✅      |    ∅    |
+| gfx803        | Polaris    | Polaris 20      | AMD Radeon™ RX 580                                |  ✅   |    ✅    |   ✅    |    ✅    |    ✅    |     ✅     |     ✅     |     ✅     |     ✅      |    ∅    |
+| gfx803        | Polaris    | Polaris 20      | AMD Radeon™ PRO 580                               |  ✅   |    ✅    |   ✅    |    ✅    |    ✅    |     ✅     |     ✅     |     ✅     |     ✅      |    ∅    |
+| gfx803        | Polaris    | Polaris 20      | AMD Radeon™ RX 570                                |  ✅   |    ✅    |   ✅    |    ✅    |    ✅    |     ✅     |     ✅     |     ✅     |     ✅      |    ∅    |
+| gfx803        | Polaris    | Polaris 20      | AMD Radeon™ PRO 570                               |  ✅   |    ✅    |   ✅    |    ✅    |    ✅    |     ✅     |     ✅     |     ✅     |     ✅      |    ∅    |
+| gfx803        | Polaris    | Polaris 30      | AMD Radeon™ RX 590                                |  ✅   |    ✅    |   ✅    |    ✅    |    ✅    |     ✅     |     ✅     |     ✅     |     ✅      |    ∅    |
+| gfx900        | Vega       | Vega 10 XL      | AMD Radeon™ RX Vega 56                            |  ✅   |    ✅    |   ✅    |    ✅    |    ✅    |     ✅     |     ✅     |     ✅     |     ✅      |    ∅    |
+| gfx900        | Vega       | Vega 10 XL      | AMD Radeon™ PRO Vega 56                           |  ✅   |    ✅    |   ✅    |    ✅    |    ✅    |     ✅     |     ✅     |     ✅     |     ✅      |    ∅    |
+| gfx900        | Vega       | Vega 10 XT      | AMD Radeon™ Instinct MI25                         |  ✅   |    ✅    |   ✅    |    ✅    |    ✅    |     ✅     |     ✅     |     ✅     |     ✅      |    ∅    |
+| gfx900        | Vega       | Vega 10 XT      | AMD Radeon™ RX Vega 64                            |  ✅   |    ✅    |   ✅    |    ✅    |    ✅    |     ✅     |     ✅     |     ✅     |     ✅      |    ∅    |
+| gfx900        | Vega       | Vega 10 XT      | AMD Radeon™ RX Vega 64 Liquid                     |  ✅   |    ✅    |   ✅    |    ✅    |    ✅    |     ✅     |     ✅     |     ✅     |     ✅      |    ∅    |
+| gfx900        | Vega       | Vega 10 XTX AIR | AMD Radeon™ Vega Frontier Edition                 |  ✅   |    ✅    |   ✅    |    ✅    |    ✅    |     ✅     |     ✅     |     ✅     |     ✅      |    ∅    |
+| gfx900        | Vega       | Vega 10 XTX LCS | AMD Radeon™ Vega Frontier Edition (Liquid Cooled) |  ✅   |    ✅    |   ✅    |    ✅    |    ✅    |     ✅     |     ✅     |     ✅     |     ✅      |    ∅    |
+| gfx906:xnack- | Radeon VII | Vega 20 GL      | AMD Radeon™ Instinct MI50                         |  ✅   |    ✅    |   ✅    |    ✅    |    ✅    |     ✅     |     ✅     |     ✅     |     ✅      |    ∅    |
```

Comment:
I see we're starting to put the xnack mode here -- we might compile some of the libraries in a particular set of xnack modes, but that doesn't really change the product names. For instance, down below in the gfx90a:xnack- and gfx90a:xnack+ lines, those the same product running in different modes.